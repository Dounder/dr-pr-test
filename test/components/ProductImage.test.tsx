import * as React from 'react';
import renderer from 'react-test-renderer';

import ProductCard, { ProductImage } from '../../src/components';
import { produc2 } from '../data/products';

describe('ProductTitle', () => {
  test('should show not image component', () => {
    const wrapper = renderer.create(
      <ProductImage img="https://test.com" className="test-class" />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  test('should show a product image', () => {
    const wrapper = renderer.create(
      <ProductCard product={produc2}>{() => <ProductImage />}</ProductCard>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
