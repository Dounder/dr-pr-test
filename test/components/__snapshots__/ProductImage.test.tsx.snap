// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle should show a product image 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductTitle should show not image component 1`] = `
<img
  alt="Product"
  className="productImg test-class"
  src="https://test.com"
/>
`;
